src/main/java/org/jenkinsci/plugins/workflow/support/storage/BulkFlowNodeStorage.java

```diff
@@ -243,6 +245,12 @@ private void storeActions() {  // We've already loaded the actions, may as well
         }
     }
 
+    @Initializer(after = InitMilestone.EXTENSIONS_AUGMENTED)
```

Comment:
I need to check when `ItemListener.resume` is called and make sure we complete `before` that.